deploy/operator.yaml

```diff
@@ -31,7 +31,8 @@ spec:
             - name: OPERATOR_NAME
               value: "mongodb-kubernetes-operator"
             - name: AGENT_IMAGE # The MongoDB Agent the operator will deploy to manage MongoDB deployments
-              value: quay.io/mongodb/mongodb-agent:10.19.0.6562-1
+              value: quay.io/mongodb/mongodb-agent-dev:scaledown
```

Comment:
this image is the same as the previous one, but just with a new readiness probe, I will create a separate PR once this is no longer a dev image.